data/macos.yaml

```diff
@@ -11,6 +11,21 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#System_Logs'
 ---
+name: MacOSAppleUnifiedLog
+doc: Apple Unified Log
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/var/db/diagnostics/*.tracev3'
+    - '/var/db/diagnostics/*/*.tracev3'
+    - '/var/db/uuidtext/*'
```

Comment:
`/var/db/uuidtext/*` will only get you the files in `/var/db/uuidtext/` you likely want `/var/db/uuidtext/*/*` as well?

Reply:
Yes, sorry it was a typo. First level is only directories.